src/lib.rs

```diff
@@ -242,6 +242,10 @@ mod joiner;
 mod keyedvec;
 #[cfg(feature = "bit-vec")]
 mod bit_vec;
+#[cfg(feature = "generic-array")]
```

Comment:
"generic-array" is not a feature in Cargo.toml

Reply:
Dependencies are automatically added as features in Cargo.